src/app/services/sort-helper-service.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, Type } from '@angular/core';
+import { Sort } from '../models/sort';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SortHelperServiceService {
```

Comment:
This should be called just **SortHelperService**; update the name of the file as well.